src/Core/Framework/Api/ApiDefinition/Generator/Schema/AdminApi/components/schemas/consentListResponse.json

```diff
@@ -0,0 +1,45 @@
+{
+  "openapi": "3.1.0",
+  "info": [],
+  "paths": [],
+  "components": {
+    "schemas": {
+      "consentListResponse": {
+        "type": "array",
+        "description": "List of consent states for the authenticated user",
+        "items": {
+          "$ref": "#\/components\/schemas\/consentState"
+        }
+      },
+      "consentState": {
+        "type": "object",
+        "description": "Represents the current state of a consent for a specific user",
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "The unique name/identifier of the consent"
+          },
+          "scope": {
+            "type": "string",
+            "description": "The scope of the consent, eg global or admin_user. 'global' means the consent applies to all users, 'admin_user' means the consent applies to the authenticated user only"
+          },
+          "identifier": {
+            "type": "string",
+            "description": "The user identifier for whom this consent state applies, if the scope is not global"
+          },
+          "status": {
+            "type": "string",
+            "enum": ["requested", "accepted", "revoked"],
+            "description": "The current state of the consent. 'requested' means the user has not yet responded, 'accepted' means the user granted consent, 'revoked' means the user withdrew consent"
+          },
+          "actorId": {
+            "type": "string",
+            "description": "The user identifier of who performed the consent action, eg for global consents, it it user who accepted/revoked it"
```

Comment:
what's here when the status is `requested`?

Reply:
`null` - I will update!